### PR TITLE
Borg usage

### PR DESCRIPTION
**Pull request by anas10**
This is related to issue #21. Improved usage and commands declaration

### CODE REVIEW

**Comment by anas10** [comment]
The summary for each command should be updated though. I'll update that later except if someone else wants to do it.

**Comment by crufter** [comment]
@anas10 can you get rid of the .gitignore files?

File: main.go [hunk 1]
```diff
@@ -13,29 +14,28 @@ func main() {
...
+		if err := c.F(flag.Args()); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
 	}
 }
 
 func help() {
-	fmt.Println("Usage: borg \"your question\"")
+	fmt.Print("\033[4mUsage:\033[0m\n\n")
```
Comment (crufter) [inline main.go:29]: This is pretty hard to read. Any way to make this better?

File: main.go [hunk 2]
```diff
@@ -13,29 +14,28 @@ func main() {
...
-	case "editor":
-		err = commands.Editor(flag.Arg(1))
-	default:
-		err = commands.Query(flag.Arg(0))
-	}
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+
+	if c, ok := commands.Commands[flag.Arg(0)]; !ok {
```
Comment (crufter) [inline main.go:18]: Why is the query command any special? So far the main.go was really easy to understand - if a bit manual - now it's hard to get what's going on.

File: main.go (hunk 1 above)
Comment (slavovojacek) [reply main.go:29]: @crufter @anas10 is this for colour highlighting or something similar? I am referring to `\033[4m`. If so, I am sure you can abstract it into a method that wraps a string around a specific colour?

**Review by slavovojacek — APPROVED** [review]

**Comment by slavovojacek** [comment]
@crufter not quite sure what you mean by "removing the .gitignore files"?
@anas10 should we add .DS_Store into .gitignore too?

File: main.go (hunk 1 above)
Comment (jeremyletang) [reply main.go:29]: yeah maybe use this kind of lib: https://github.com/fatih/color,
and enable color with a flag.

**Pull request merged**